README.md

```diff
@@ -88,3 +88,32 @@ gauge('test').set_value(42)
 ```
 
 See `examples/pyformance_usecase.py` for a complete code example using Pyformance.
+
+## Known Issues
```

Comment:
Not sure if this is called Known Issues or Release Notes. Or do we even out this here.